conan/cli/command.py

```diff
@@ -75,7 +75,7 @@ def _fill_text(self, text, width, indent):
 
 class BaseConanCommand(object):
     def __init__(self, method, formatters=None):
-        self._formatters = {}
+        self._formatters = {"text": lambda x: None}
```

Comment:
Maybe it is better to leave it empty, and if something returns something but ``text`` is not implemented, then fail?

Reply:
I made it that way because there are cases that you don't have a text formatter but you do have a json formatter, so you have to return data but the text formatter will just do nothing

Reply:
Ok, makes sense 👍